osu.Game.Tests/Visual/Online/TestSceneUserProfileOverlay.cs

```diff
@@ -106,6 +106,14 @@ protected override void LoadComplete()
                 Country = new Country { FullName = @"Japan", FlagName = @"JP" },
                 CoverUrl = @"https://osu.ppy.sh/images/headers/profile-covers/c6.jpg"
             }, api.IsLoggedIn));
+            AddStep("Show bancho", () => profile.ShowUser(new User
```

Comment:
CI's not complaining, but for consistency with the other steps and to separate them out visually I'd add a blank line before this.